### PR TITLE
WuYang first example finished.

### PR DESCRIPTION
**Pull request by ymshi449**
I showed examples for WuYang method and visualization.

### CODE REVIEW

**Comment by ymshi449** [comment]
I further updated an example for WY regularization.

File: WuYang/Noble_Gases.ipynb [hunk 1]
```diff
@@ -42,91 +51,191 @@
     "symmetry c1\n",
     "\"\"\" )\n",
     "\n",
-    "psi4.set_options({\"reference\" : \"rhf\"})\n",
-    "e, wfn = psi4.energy(\"ccsd/aug-cc-pvtz\", return_wfn=True, molecule=Be)\n",
-    "\n"
+    "psi4.set_options({\"reference\" : \"rhf\"})  # Spin-Restricted\n",
+    "\n",
+    "# IMPORTANT NOTE: psi4.energy does not update cc densities. So we calculate dipole momentes instead.\n",
```
Comment (VHchavez) [inline WuYang/Noble_Gases.ipynb]: "momentes" -> "moments"

**Review by VHchavez — COMMENTED** [review]
This looks good. We may change a lot of things along the way. But having these already in the repo will easily make me understand the new grid functions. 
So I think we should merge this as well.

**Comment by VHchavez** [comment]
Can we add links to the papers cited? Would be nice to just be able to click them and be directed to them.

**Comment by ymshi449** [comment]
Added

**Pull request merged**